### PR TITLE
Fix/named upload

### PR DESCRIPTION
**Pull request by awitherow**
Closes https://github.com/mortik/fastlane-plugin-raven/issues/2

This is still work in progress at the moment @mortik.

What are your thoughts on the WIP commit and the general implementation?

### CODE REVIEW

File: lib/sentry/client.rb [hunk 1]
```diff
@@ -34,10 +34,15 @@ def create_release(version)
...
     def upload_release_artifact(version, artifact_path)
+      # fileName = get artifact_path file.
+      # relative_path = get relative_path variable
       Typhoeus.post(
         "#{base_url}/#{slug}/releases/#{version}/files/",
         headers: { "Authorization" => "Bearer #{token}" },
-        body: { file: File.open(artifact_path, "r") }
+        body: {
+          file: File.open(artifact_path, "r"),
+          name: # #{relative_path}/#{fileName}
```
Comment (mortik) [inline lib/sentry/client.rb]: we could get the filename from the artifact_path like ```File.basename(artifact_path)```

Comment (awitherow) [inline lib/sentry/client.rb]: Nice! 👍

**Comment by awitherow** [comment]
@mortik apparently react native serves its code via a spawned localhost server so we need to figure out a way to properly implement the name with that relative location.

**Comment by mortik** [comment]
Yes i know... RelativePath does not fit this exactly... are you sure that the filename needs to include the local server? The Sentry Documentation does not cover this sadly.

**Comment by mortik** [comment]
@awitherow lets split this up into a two step approach... first lets fix the missing name param with the file basename and check if this solves our problem if not we can revisit the idea of adding a name prefix.

**Comment by awitherow** [comment]
@mortik this should be enough?

**Pull request merged**